aspnet/visual-studio/overview/2013/aspnet-and-web-tools-20132-preview-for-visual-studio-2013-release-notes.md

```diff
@@ -257,7 +256,7 @@ Entity Framework has been updated to version 6.1 for both runtime and tooling. E
 <a id="owin"></a>
 ### Microsoft OWIN components
 
-There were lots of bugs that were fixed in this release. Please see the [release notes for the 2.1.0 release](https://katanaproject.codeplex.com/releases/view/113281) for more detailed information.
+There were lots of bugs that were fixed in this release. 
```

Comment:
I will attempt to find some patch notes. There were a lot of bugs that were fixed in this release.